packages/manager/modules/pci/src/projects/project/storages/databases/components/order-command/translations/Messages_en_GB.json

```diff
@@ -1,15 +1,15 @@
 {
   "pci_database_order_command_title": "Equivalent order",
-  "pci_database_order_command_description": "The following command allows you to create your database using",
+  "pci_database_order_command_description": "The following command allows you to create your service using",
   "pci_database_order_command_api_link": "our API",
   "pci_database_order_command_parameters": "Settings:",
   "pci_database_order_command_parameters_info": "More information on the available settings:",
-  "pci_database_order_command_parameters_info_description": "Your database name",
+  "pci_database_order_command_parameters_info_description": "Your service name",
```

Comment:
For translations, we only add/edit text for FR (French). The translations for the rest of the languages are to be generated via a translation request